services.yaml

```diff
@@ -3078,6 +3105,42 @@ definitions:
         items:
           type: string
           example: MyApp
+  JVM:
+    description: JVM properties
+    type: object
+    properties:
+      name:
+        type: string
+        description: JVM name
+        example: prod-api
+      deployment:
+        type: string
+        description: deployment name
+        example: v4.0.2041
+      PID:
+        type: string
+        description: PID
+        example: 20187
+      agent_version:
+        type: string
+        description: Version of the agent installed on the JVM
+        example: 4.18.1
+      collector_version:
+        type: string
+        description: Overops Collector version
+        example: 4.18.2
+      status:
+        type: string
+        enum: [monitored, OverOps not attached]
```

Comment:
there are more status and i wouldn't use an enum here but rather an open list of strings